src/IOBuffer.ts

```diff
@@ -79,6 +80,12 @@ export class IOBuffer {
       this.lastWrittenByte = data.byteLength;
     }
 
+    this.hostBigEndian = (() => {
+      const array = new Uint8Array(4);
+      const view = new Uint32Array(array.buffer);
+      return !((view[0] = 1) & array[0]);
+    })();
```

Comment:
You can extract this to a variable at the top-level. There is no reason (I think) for this value to be different between instances of the class.